tests/scripts/expect/tun-realm-local-multicast.exp

```diff
@@ -113,6 +113,7 @@ wait_for "state" "child"
 set mleid4 [get_mleid]
 
 switch_node 1
+sleep 10
```

Comment:
Always pass during local ~ 60 runs.

Reply:
This is fine as a quick fix. But should we improve the performance of OT because 10s without reachability seems to be an issue?

Reply:
See the discussion in https://github.com/openthread/openthread/issues/5773#issuecomment-722865993
Feel free to share your thoughts there.